tools/driver/CMakeLists.txt

```diff
@@ -13,6 +13,21 @@ target_link_libraries(swift-frontend
                         swiftSymbolGraphGen
                         LLVMBitstreamReader)
 
+# Generate and install supported feature JSON file to the toolchain.
+if(NOT CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
Why not-windows?  This condition seems wrong.  If this was due to the incorrect use of `touch`, this should be `if(NOT CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)`, you could be building for Windows on macOS ;-)

Reply:
Yeah, we should use `CMAKE_HOST_SYSTEM_NAME` here.

Reply:
I think that question of why not-windows still remains.  What makes SupportedFeatures.json not applicable to Windows?